### DISCUSSION TITLE
UC2 Incubator Fluorescence Scope

### DISCUSSION THREAD

**Discussion by wbrett87** [bionanoimaging/UC2-GIT#84] [Q&A]
Greetings,

I've begun printing UC2 cubes with the end goal of making an incubator fluorescence microscope. I'm a research technician and soon to be (hopefully) PhD student. My lab is interested in tracing developmental expression of a gene we suspect has to do with ciliogenesis in tracheal multi ciliated cells. At the moment we are not equipped to do long time lapse videos and would benefit greatly from being able to do so. It seems to me that there are two versions of the incubator fluorescence scope: the basic and the infinity corrected version. Should I start with the basic or go straight to building the infinity optics version. What are the pros and cons of each?

In addition, it would be to eventually image in multiple colors. Could I adapt the scope to image in Green(EGFP) and Red(mScarlet)?

I'd also like to replace the raspicamv2/phone camera with the new arducam 64megapixel camera that they are releasing at the end of may. Does this sound feasible or would it require a large modification.

Thanks again for any help you can give!

![IMG_2086](https://user-images.githubusercontent.com/103341754/162643724-6bac47c3-5ee2-4a5d-89c5-d06fe4dc9bfe.JPG)

**Comment by beniroquai** [comment]
Hey @wbrett87, many apologies for my late reply. "Discussions" are a new thing ;-) 

First of all: Great that you decided the quest on building a UC2 system. What you are trying to do is very challenging for the first project I must admit. Especially since you are trying to have two colors simultaneously. But there is help on the way.  
We have one additional repository for an [inverted fluorescence microscope](https://github.com/openUC2/UC2_Fluorescence_microscope), which has been build multiple times. However, since we have the injection molded cubes, we rely on those mostly - stability in the incubator is definitely an issue with the 3D printed ones. 

@ranranking and I also build a filter switcher for selecting different laser wavelengths. We still need to document this properly. Good point, I should do that [here](https://github.com/openUC2/UC2-MultiColour). 

If you decide to build it, I would suggest to somehow collaborate since some of the important aspects have not yet been documented properly. And we need to do that for sure! 
For the first experiment, I would suggest to start with the brightfield transmission microscope. Once you have your fist image, you can iterate. 
For the fluorescence microscope, you would also need another camera, like those from Daheng Imaging with a monochrome sensor.

**Reply by wbrett87** [reply]
No problem, the world is a busy place and I understand fully! 

I will definitely take your advice and build the brightfield scope first. Should I build this with 3d printed cubes or go straight to IM? If I want to go the IM route, would you be able to sell me some?

As far as the camera goes , what is the benefit of a monochrome camera? Is it impossible to use a color cam? In the interests of cost savings, here is a 10mp mono camera from Arducam  ([here](https://www.arducam.com/product/arducam-cmos-mt9j001-1-2-3-inch-10mp-monochrome-camera-module/) that only costs $50. Would this work? The only issue I can see is that it is a dev board and probably doesn't have a good software interface. I don't know how difficult it would be to develop one, but would be willing to try and see if I can figure it out. It looks like this sensor is used in some plug and play microscope cameras already on sale.

Thanks for getting back to me!

**Reply by beniroquai** [reply]
Hey, we will be able to produce and potentially sell IM cubes soon - and yes of course those are much better in terms of stability ;-)

In terms of camera; You want to have a low noise level (i.e. high signal to noise) that means: low readnoise/darknoise, etc. and high quantum efficiency (i.e. photon to electron conversion). The Raspberry Pi camera is not bad at all but comes with a number of drawbacks (e.g. non-uniform microlens distribution see [https://openhardware.metajnl.com/articles/10.5334/joh.20/](https://openhardware.metajnl.com/articles/10.5334/joh.20/)). RGB filters block up to 75 of the light in case of monocrhomatic fluorescence, thats something you don't want of course.. Industry-grade cameras that do a good job can be found form daheng: https://www.get-cameras.com/ We fully support them inside the Imswitch software: https://github.com/beniroquai/ImSwitch/

**Reply by beniroquai** [reply]
Some information about ImSwitch + UC2: https://youtu.be/7f1XH7lQcgQ?t=3410
